docs/contracts.rst

```diff
@@ -485,7 +485,9 @@ Functions can be declared ``view`` in which case they promise not to modify the
 .. note::
   If the compiler's EVM target is Byzantium or newer (default) the opcode
   ``STATICCALL`` is used for ``view`` functions which enforces the state
-  to stay unmodified as part of the EVM execution.
+  to stay unmodified as part of the EVM execution. For library ``view`` functions
+  ``DELEGATECALL`` is used. Since they are executed in the context of the calling
```

Comment:
perhaps add: is used, because there is no combined delegatecall and staticcall. This means library view functions do not have run-time checks that prevent state modifications. This is not a big problem because library code is usually known at compile-time and the static checker performs compile-time checks.